standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -9483,6 +9483,81 @@ private void dropPartitionColumnStatisticsNoTxn(
     queryWithParams.getLeft().closeAll();
   }
 
+  @Override
+  public void deleteAllPartitionColumnStatistics(TableName tn, String writeIdList) {
+
+    String catName = tn.getCat();
+    String dbName = tn.getDb();
+    String tableName = tn.getTable();
+
+    Query query = null;
+    dbName = org.apache.commons.lang3.StringUtils.defaultString(dbName, Warehouse.DEFAULT_DATABASE_NAME);
+    catName = normalizeIdentifier(catName);
+    if (tableName == null) {
+      throw new RuntimeException("Table name is null.");
+    }
+    boolean ret = false;
+    try {
+      openTransaction();
+      MTable mTable = getMTable(catName, dbName, tableName);
+
+      query = pm.newQuery(MPartitionColumnStatistics.class);
+
+      Object engine = null;
```

Comment:
This seems strange. Setting this to null and one line below checking

Reply:
oh..sorry; I plan to disable this option by default - and leave only a targeted test to cover for it
but I wanted to make a sanity check that this will work in case all tests are executed (and it does)

I should have marked it as wip or something..I'll clean up things like this in the next version. 

thank you for taking a look!  I was testing with psql so far but  I'll check mysql as well!